### DISCUSSION TITLE
favorite Emotes wheel/QOL idea

### DISCUSSION THREAD

**Discussion by salemas5** [Friendly0Fire/GW2Radial#332] [Ideas]
how about adding new emote wheel. And ability to switch from one wheel to the other with mouse left/right.  Something in the realm how Overwatch do it.

**Comment by Friendly0Fire** [comment]
I can see the appeal, but it'd be a significant effort since it touches the chat window which has its own can of worms associated with it. Further, it *technically* breaks the "one input one output" rule and is gameplay relevant in a handful of situations, so I'd have to consider that angle (not that I'm really worried, but it's worth pointing out).

I'd welcome a PR for it, but I probably won't be implementing this myself.
